audb/core/load.py

```diff
@@ -810,7 +810,15 @@ def _load_files(
 def _misc_tables_used_in_scheme(
     db: audformat.Database,
 ) -> typing.List[str]:
-    r"""List of misc tables that are used inside a scheme."""
+    r"""List of misc tables that are used inside a scheme.
+
+    Args:
+        db: database object
+
+    Returns:
+        unique list of misc tables used in schemes
```

Comment:
Should this then not a `set` in order to be straightforward?

Reply:
I find working with lists easier, that's the reason why I always prefer to return a list. The only exception is, if I really need to use set specific operations, but this is not the case here.

Reply:
Ok fair enough, so you do not have to do any list conversion of the set by the caller. Then I would resolve this conversation.